src/core/store/modules/lines/lines.reducer.ts

```diff
@@ -177,6 +180,21 @@ const linesReducer = createReducer(linesInitialState, (builder) => {
     // .addCase(initiateSaveLines.rejected, (state, { error }) => {
     //   state.statusMap.initiateSaveLines = { error: error.message };
     // })
+    /* --------------------------- getInterestAccrued ------------------------- */
```

Comment:
can u do something about all this commented out code. We have it for everything else so best to stay consistent and have the pending and error state filled

Reply:
done